src/OrchardCore.Modules/OrchardCore.Contents/Services/IQueryFromFilterBox.cs

```diff
@@ -0,0 +1,17 @@
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Records;
+using OrchardCore.Contents.ViewModels;
+using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.Navigation;
+using System.Threading.Tasks;
+using YesSql;
+
+namespace OrchardCore.Contents.Services
+{
+    public interface IQueryFromFilterBox
```

Comment:
I don't think we need an interface, we can directly register and resolve the class, no need for an abstraction. Even more if this is not in the abstractions project.

Same for ISyerContentTypesProvider. Just merge the two classes into one.

Reply:
You resolve the class itself, like you were doing before with the interface.